package.json

```diff
@@ -30,24 +31,30 @@
     "artsy-newrelic": "artsy/artsy-newrelic",
     "artsy-xapp": "^1.0.4",
     "async": "^1.4.2",
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.25.0",
-    "babel-loader": "^7.1.2",
+    "@babel/core": "7.0.0-beta.42",
+    "@babel/cli": "7.0.0-beta.42",
+    "@babel/node": "7.0.0-beta.42",
+    "@babel/plugin-proposal-class-properties": "7.0.0-beta.42",
+    "@babel/plugin-transform-runtime": "7.0.0-beta.42",
+    "@babel/plugin-transform-typescript": "7.0.0-beta.42",
+    "@babel/preset-env": "7.0.0-beta.42",
+    "@babel/preset-es2015": "7.0.0-beta.42",
+    "@babel/preset-react": "7.0.0-beta.42",
+    "@babel/preset-stage-3": "7.0.0-beta.42",
+    "@babel/preset-typescript": "7.0.0-beta.42",
+    "@babel/register": "7.0.0-beta.42",
+    "@babel/runtime": "7.0.0-beta.42",
+    "@babel/template": "7.0.0-beta.42",
+    "@babel/types": "7.0.0-beta.42",
+    "babel-loader": "8.0.0-beta.0",
     "babel-plugin-inline-react-svg": "^0.4.0",
-    "babel-plugin-module-resolver": "^2.7.1",
+    "babel-plugin-module-resolver": "^3.0.0",
+    "babel-plugin-react-transform": "^3.0.0",
     "babel-plugin-rewire": "^1.1.0",
-    "babel-plugin-styled-components": "^1.1.7",
-    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-preset-react-hmre": "^1.1.1",
+    "babel-plugin-styled-components": "^1.4.0",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
```

Comment:
I doubt these are meant for Babel 7 (e.g. doesn’t start with `@babel/`) but moreover the right plugins for these features should already be enabled by the various `@babel/preset-*` packages above, so this can just be removed (as well as from `.babelrc`).